components/Footer/Footer.tsx

```diff
@@ -69,7 +69,7 @@ const Footer: React.FC<FooterProps> = ({ className }) => {
         className || ""
       } footer`}
     >
-      <div className="md:w-[1280px] w-[90%]">
+      <div className="w-[90%] xl:w-[1280px] xl:!max-w-[1280px]">
```

Comment:
looks amazing

Do you need the `:!max-w`? defining both breakpoints should work without. right?

as is, looks awesome